test/lib/ansible_test/_data/collection_detail.py

```diff
@@ -57,6 +57,8 @@ def main():
 
     try:
         result = read_manifest_json(collection_path) or read_galaxy_yml(collection_path)
+        if result is None:
+            result = dict()
```

Comment:
A less verbose way to write this:

```suggestion
        result = read_manifest_json(collection_path) or read_galaxy_yml(collection_path) or {}
```

Reply:
Thanks, fixed it.